sd_dynamic_prompts/ui/wildcards_tab.py

```diff
@@ -226,6 +225,6 @@ def save_file_callback(js):
             with path.open("w") as f:
                 contents = contents.splitlines()
                 for c in contents:
-                    f.write(c.strip() + os.linesep)
+                    f.write(c.strip() + "\n")
```

Comment:
I think
```python
print(c.strip(), file=f)
```
would also be acceptable :)

Reply:
Thanks - I prefer your suggestion. Seems strange that the docs say to use "\n" instead of os.linesep. Your suggestion better abstracts the nuances of different platforms.